perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -711,18 +711,26 @@ sub bin_services_for_address {
     my $skip_bags = $self->{c}->get_param('skip_bags');
 
     @out = () if $bags_only;
-    my $food_containers = $bags_only ? [ 428 ] : $skip_bags ? [ 424, 423 ] : [ 424, 423, 428 ];
 
-    push @out, {
+    my @food_containers;
+    if ($bags_only) {
+        push @food_containers, 428;
+    } else {
+        push(@food_containers, 424) unless $open_requests->{424}; # Large food caddy
```

Comment:
Could this and the next line also check for open 493 as that's a both bins request?